.gitignore

```diff
@@ -0,0 +1,2 @@
+inventory/group_vars/kube-master
```

Comment:
no need to add these to .gitignore

Reply:
Definately not. These should be there as a template for the user to know what vars to pass to ansible.